docs/providers.md

```diff
@@ -9,44 +9,44 @@ Supported providers
+{% include callout.html content="While OpenID providers do not require an application [[1]](http://openid.net/specs/openid-connect-core-1_0.html#Overview) and thus no action from your part to make their adapters work, OAuth providers requires consumer (or client) websites (i.g., your website) to register an application [[2]](http://tools.ietf.org/html/rfc5849#page-3) [[3]](http://tools.ietf.org/html/rfc6749#section-2). Generally speaking those providers will have a dedicated developer's section or subdomain where you can register yours and apply for a pair of keys (i.e, application credentials )." type="primary" %}
 
 Adapter Unique Name | Specs   | Authentication | User's Profile | User's Contacts | User's Status  | User's Activity Stream
 ------------------- | ------- | -------------- | -------------- | --------------- | -------------- | ----------------------
-Facebook            | OAuth2  | [X]            | [X]            | [X]             | [X]            | [X]                  
-Twitter             | OAuth1  | [X]            | [X]            | [X]             | [X]            | [X]                  
-Google              | OAuth2  | [X]            | [X]            | [X]             |                |                      
-GitHub              | OAuth2  | [X]            | [X]            |                 |                |                      
-GitLab              | OAuth2  | [X]            | [X]            |                 |                |                      
-Reddit              | OAuth2  | [X]            | [X]            |                 |                |                      
-BitBucket           | OAuth2  | [X]            | [X]            |                 |                |                      
-WordPress           | OAuth2  | [X]            | [X]            |                 |                |                      
-Tumblr              | OAuth1  | [X]            | [X]            | [X]             |                |                      
-Disqus              | OAuth2  | [X]            | [X]            |                 |                |                      
-Dribbble            | OAuth2  | [X]            | [X]            |                 |                |                      
-WindowsLive         | OAuth2  | [X]            | [X]            | [X]             |                |                      
-Foursquare          | OAuth2  | [X]            | [X]            | [X]             |                |                      
-Instagram           | OAuth2  | [X]            | [X]            |                 |                |                      
-LinkedIn            | OAuth2  | [X]            | [X]            |                 |                |                      
```

Comment:
hi, I am new user. I have LnkedIn appid, Secret and AccessToken in my DB. How can I post a status by using setUserStatus() 
nb: though I have all those data so I don't want to push user in login again. Is it possible to post inLinkedIn by using this method.?

Reply:
Hi,
Check comment here: https://github.com/hybridauth/hybridauth/issues/888#issuecomment-354552714

Reply:
Hi,
Thanks for your suggestion. But I am getting new issue. Please please help me!
Here is my method to post in LinkedIn but I am getting error : Fatal Error: 
ArrayObject Object
(
    [storage:ArrayObject:private] => Array
        (
            [exception] => Hybridauth\Exception\HttpRequestFailedException Object
                (
                    [message:protected] => Signed API request has returned an error. HTTP error 401. Raw Provider API response: {
  "errorCode": 0,
  "message": "Empty oauth2_access_token",
  "requestId": "GT3BPWBLSX",
  "status": 401,
  "timestamp": 1520772798106
}.
//This is my method........
 public function postToLinkedin($lnAppkey,$lnAppSec,$lnToken)
 {
         if($lnAppkey !=null && $lnAppSec !=null && $lnToken !=null)
        {
            $_SESSION['app_id_linkedIn'] = $lnAppkey;
            $_SESSION['app_sec_linkedIn'] = $lnAppSec;
            $_SESSION['access_token_linkedIn'] = $lnToken;
        }

        $config = [
            'callback' => $request->getBaseUrl().'/socialpost/application/postToLinkedin',
            "enabled" => true,
            'keys' => [
                'key' => $_SESSION['app_id_linkedIn'],
                'secret' => $_SESSION['app_sec_linkedIn']
            ],

        ];
// Instantiate LinkedIn Adapter.
        $linkedin = new Hybridauth\Provider\LinkedIn($config);

// Authenticate the user.
        $linkedin->setAccessToken($_SESSION['access_token_linkedIn']);

// Update the user status.
        $response = $linkedin->setUserStatus([
            'comment' => '',
            'content' => [
                'title' => 'This is a post',
                'description' => "I am trying to post a demo content",
                'submitted-url' => '',
                'submitted-image-url' => '',
            ],
            'visibility' => [
                'code' => 'anyone',
            ],
        ]);
        
 }

Reply:
Only one question, why here? why you add those comments in merged PR. Pls fill the issue if you have any troubles. 🤦‍♂️